documentation/modules/upgrading/proc-upgrade-brokers-newer-kafka.adoc

```diff
@@ -50,15 +48,17 @@ spec:
       # ...
 ----
 +
+If `log.message.format.version` and `inter.broker.protocol.version` are not configured, Strimzi updates these versions automatically following the update to the Kafka version.
```

Comment:
_Suggested change:_

If `log.message.format.version` and `inter.broker.protocol.version` are not configured, Strimzi automatically updates these versions to the current defaults after the update to the Kafka version in the next step.